### PR TITLE
[Doc] Pandas, Parrow, DuckDB integration

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: docs/src/integrations.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+df = table.search(query_vector).limit(1).to_df()
+print(df)
+```
+
+```
+    vector     item  price        score
+0  [5.9, 26.5]  bar   20.0  14257.05957
+```
+
+We can apply the filter via `LancdDB`, or apply the filter on `pd.DataFrame` later.
```
Comment (changhiskhan) [inline docs/src/integrations.md:53]: LanceDB

Comment (changhiskhan) [reply docs/src/integrations.md:53]: "If you have a simple filter, it's faster to provide a `where` clause to LanceDB's search query. If you have more complex criteria, you can always apply the filter to the resulting pandas DataFrame from the search query."

File: docs/src/integrations.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+# Integrations
+
+Built on top of Apache Arrow, `LanceDB` is easy to integrate with the Python ecosystem, including Pandas, PyArrow and DuckDB.
+
+## Pandas and PyArrow
+
+``` sh
+pip install pandas lancedb
```
Comment (changhiskhan) [inline docs/src/integrations.md:8]: this should be covered in the main section already.

File: docs/src/integrations.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+```
+
+First, we need to connect to a LanceDB instance. This instance could be a local file directory.
```
Comment (changhiskhan) [inline docs/src/integrations.md:11]: connect, create_table => basics section

create_index => indexing section

File: docs/src/integrations.md (hunk 1 above)
Comment (eddyxu) [reply docs/src/integrations.md]: fixed.

File: docs/src/integrations.md (hunk 2 above)
Comment (eddyxu) [reply docs/src/integrations.md]: ok, removed.

File: docs/src/integrations.md (hunk 3 above)
Comment (eddyxu) [reply docs/src/integrations.md]: wdym

**Review by changhiskhan — APPROVED** [review]

**Pull request merged**